src/wrappers/themis/wasm/README.md

```diff
@@ -44,6 +44,48 @@ themis.initialized.then(function() {
 ```
 Success!
 
+### Building
```

Comment:
Generally we prefer to keep detailed build instructions on the separate [doc server](https://docs.cossacklabs.com/pages/documentation-themis/#webassembly-wrapper-installation) just so that we don’t have to update them in multiple places.

This readme is also [visible on npm](https://www.npmjs.com/package/wasm-themis) and users typically don’t need to be concerned with building from source, `npm install wasm-themis` should be enough.

So I believe we don’t have to make any changes in this file.

Reply:
You can ignore the change and that's OK for me. I just think it is more convenient if README.MD include build guide and there is no need to search it.

Reply:
Well, if you ask me then I’d also prefer to see build instructions somewhere on GitHub (maybe not in README, but in BUILDING.md or something).

  I’d defer this to the person maintaining the docs. @karen, what do you think? Should be put the build instructions here? Or maybe put a direct link from README to build instructions on the doc server?

Reply:
We were struggling with supporting documentation always up-to-date, as we found out that sometimes we forgot to update documentation docs / readme docs / installation docs, if they are placed in different locations (doc server, Themis wrapper folders, example folders, and so on).

Currently we prefer to keep one "source of truth" – the DocServer, and put links everywhere to it. Downsides is that users will need to follow links to external resource, but they will always find updated docs.

My suggestion is to place two links instead of copy-pasting the text.

 ```suggestion
### Building

You can install [wasmthemis from npm](https://docs.cossacklabs.com/pages/js-wasm-howto/) or [build from sources](https://docs.cossacklabs.com/pages/documentation-themis/#webassembly-wrapper-installation).
```